includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -271,13 +272,16 @@ public function update_intention(
 		$currency_code,
 		$save_payment_method = false,
 		$customer_id = '',
+		$metadata = [],
 		$level3 = [],
 		$selected_upe_payment_type = ''
 	) {
 		$request = [
-			'amount'   => $amount,
-			'currency' => $currency_code,
-			'level3'   => $level3,
+			'amount'      => $amount,
+			'currency'    => $currency_code,
+			'metadata'    => $metadata,
+			'level3'      => $level3,
+			'description' => $this->get_intent_description( $metadata['order_id'] ?? 0 ),
```

Comment:
Nice catch on the disappeared description, @bborman22! Since I noticed this was mentioned somewhere lost in one of the threads on Slack, since fixing it only involved adding this line of code, and since there's no time like the present (and who knows when we'll next have a window to look back on some of these issues), I fixed it here if you wouldn't mind taking one more quick look!

Reply:
Awesome, I wasn't 100% sure how much effort it would be, so thanks for getting that fix in there!